packages/aws-cdk/lib/api/cxapp/exec.ts

```diff
@@ -74,7 +79,7 @@ export async function execProgram(aws: SdkProvider, config: Configuration): Prom
 
   debug('env:', env);
 
-  await exec();
+  await exec(commandLine.join(' '));
```

Comment:
Now that we use `join` here, we can probably get rid of the `appToArray()` call above, right? No point in creating an Array just to join it to a string a few lines later.

Reply:
discussed offline, leaving due to the call to `guessExecutable()`

Reply:
Never mind, `guessExecutable()` actually does some things with the array... We could probably simplify it, but I don't want to get into it in this change, which already took a long time.